test cases/common/62 exe static shared/subdir/exports.h

```diff
@@ -0,0 +1,15 @@
+#ifndef __MESON_EXPORTS__
```

Comment:
The C and C++ standards reserve all preprocessor tokens that start with one or two underscores to the stdlib, so this is undefined behaviour. Please rename or, better yet, use `pragma once`.